test-harness/specs/security-scheme-validation-AC-2.oas3.txt

```diff
@@ -29,4 +29,4 @@ curl -i http://localhost:4010/todos
 HTTP/1.1 401 Unauthorized
 www-authenticate: Basic realm="*"
 
-{"type":"https://stoplight.io/prism/errors#UNAUTHORIZED","title":"Invalid security scheme used","status":401,"detail":"","headers":{"WWW-Authenticate":"Basic realm=\"*\""}}
+{"type":"https://stoplight.io/prism/errors#UNAUTHORIZED","title":"Invalid security scheme used.","status":401,"detail":"","headers":{"WWW-Authenticate":"Basic realm=\"*\""}}
```

Comment:
@XVincentX, do you think we are OK to leave `headers` in the response body? Started wondering... I could possibly remove that in https://github.com/stoplightio/prism/pull/514 I don't think having `headers` in the response body violates https://tools.ietf.org/html/rfc7807, but now I'm thinking: do we need this if we already have this info in the headers?

Reply:
Good question! I was wondering the same and after a while I came up with these conclusions:

1. The RFC has indeed space for some other fields, so we're still in line with it
2. Effectively the mapping between the `header` field and HTTP Headers is the same, but in theory another protocol could use the same information to return something different (I'm thinking about GraphQL or something). Moreover, The HTTP Client could use these informations to show something to the UI (or further elaboration) The point is, this is not directly related to the HTTP Server.